package.json

```
@@ -32,7 +32,7 @@
   },
   "dependencies": {
     "chalk": "^2.1.0",
-    "hippo-js": "https://github.com/deislabs/hippo-client-node/releases/download/v0.0.1/hippo-js-0.0.1.tgz",
+    "hippo-js": "https://github.com/deislabs/hippo-client-node/releases/download/0.0.2/hippo-js-0.0.2.tgz",
```

I have changed the release number for consistency with v0.0.1. Apologies for the screw up.
 
```suggestion
    "hippo-js": "https://github.com/deislabs/hippo-client-node/releases/download/v0.0.2/hippo-js-0.0.2.tgz",
```